pages/faq/troubleshooting/faq.md

```diff
@@ -138,6 +139,10 @@ Discontinued devices will no longer be actively supported by {{ $names.company.l
 
 There are a few options for devices that do not have an official device type on {{ $names.company.lower }}. If your device has an x86 architecture, you can try the [Intel NUC][nuc] image, which is built to support generic x86 devices. For other devices, you can [build your own][build-your-own] version of {{ $names.os.lower }} using our [open source repos][balenaOS]. To discuss custom board support, please contact sales@{{ $names.email_domain }}.
 
+##### What to keep in mind while buying or using power supply units?
+
+Power supply unit (PSU) are a critical component to any widespread productions deployment as mentioned in our [Going to Production][go-to-production] guide. When buying a PSU for your hardware, do carefully inspect and understand the power requirements of the board. Current requirements are just as important as the voltage requirements. A PSU that provides a voltage that is higher than required may damage the board, but a supplied voltage that is too low will probably cause the board to malfunction or fail to boot. If the PSU isn't capable of providing the maximum current that the board demands, then the board might boot fine but eventually fail to perform under CPU-intensive loads. As the load increases, more current is drawn than what the PSU is capable of, which leads to brown-outs (i.e. the board unexpectedly resetting). Proper research when buying PSUs or using the official adapters is recommended.
```

Comment:
```suggestion
Power supply units (PSUs) are a critical component to any widespread productions deployment as mentioned in our [Going to Production][go-to-production] guide. When buying a PSU for your hardware, carefully inspect and understand the power requirements of the board. Current requirements are just as important as the voltage requirements. A PSU that provides a voltage that is higher than required may damage the board, but a supplied voltage that is too low will probably cause the board to malfunction or fail to boot. If the PSU isn't capable of providing the maximum current that the board demands, then the board might boot fine but eventually fail to perform under CPU-intensive loads. As the load increases, more current is drawn than what the PSU is capable of, which leads to brown-outs (i.e. the board unexpectedly resetting). Proper research when buying PSUs or using the official adapters is recommended.
```

Reply:
Also change "productions deployment" to "production deployment" ?

I dunno what level of technical audience this is aimed at, but perhaps it might be worth clarifying "Current requirements (measured in Amps) are just as important as the voltage requirements (measured in Volts)." ?

"than what the PSU is capable of" might sound nicer if reworded as "than the PSU is capable of supplying" ?

_Might_ be worth mentioning that brown-outs can sometimes cause filesystem corruption?

Perhaps the "If using any peripherals, the power requirements will be higher than the board's rating." from the other page is worth reiterating / rewording here too? :man_shrugging:

Reply:
Added all the points, except for the last one. The point is mostly true but feels dependent on case to case basis. The power that can be supplied by the PSU should always be a little higher than what's needed by the board is a well-recommended rule both in IoT and building computers but I think we should leave this on the last line where they just use the official adapters or do the due diligence when buying.